index.html

```diff
@@ -1,12 +1,15 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta name="color-scheme" content="light dark" />
-        <title>Adaptive Palette</title>
-    </head>
-    <body>
-        <div id="app"></div>
-    </body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="color-scheme" content="light dark" />
+    <title>Adaptive Palette</title>
+  </head>
+  <body>
+    <h2>Palette Based on JSON</h1>
+    <div id="paletteCell">
```

Comment:
This div is used to render the entire palette. It might be better to use an id "palette".

Reply:
The `paletteCell` id was left over from when the code only rendered a single `PaletteCell`.  I have changed it to `bmwKeyCodes` for now, but I would not be surprised if it's changed again.

I also cleaned up `Palette.ts` by removing the rendering code and moving it to `index.html`.